Build/Scripts/runTests.sh

```diff
@@ -316,10 +317,11 @@ CONTAINER_INTERACTIVE="-it --init"
 HOST_UID=$(id -u)
 HOST_PID=$(id -g)
 USERSET=""
-SUFFIX=$(echo $RANDOM)
+SUFFIX="$RANDOM"
```

Comment:
Why this change? If this is to fix any warning, please let's fix it a separate PR first. Ditto for line 424/426.